gdal/swig/python/trimmedsysconfig.py

```diff
@@ -0,0 +1,75 @@
+# Trimmed version of /usr/lib/python3.8/distutils/sysconfig.py
+
+"""Provide access to Python's configuration information.  The specific
+configuration variables available depend heavily on the platform and
+configuration.  The values may be retrieved using
+get_config_var(name), and the list of variables is available via
+get_config_vars().keys().  Additional convenience functions are also
+available.
+
+Written by:   Fred L. Drake, Jr.
+Email:        <fdrake@acm.org>
+"""
+
+import os
+import sys
+
+# These are needed in a couple of spots, so just compute them once.
+PREFIX = os.path.normpath(sys.prefix)
+EXEC_PREFIX = os.path.normpath(sys.exec_prefix)
+BASE_PREFIX = os.path.normpath(sys.base_prefix)
+BASE_EXEC_PREFIX = os.path.normpath(sys.base_exec_prefix)
+
+def get_python_version():
+    """Return a string containing the major and minor Python version,
+    leaving off the patchlevel.  Sample return values could be '1.5'
+    or '2.2'.
+    """
+    return '%d.%d' % sys.version_info[:2]
+
+
+
+def get_python_lib(plat_specific=0, standard_lib=0, prefix=None):
```

Comment:
Since GDAL already provides the prefix, some of the code in this function could be removed.

Reply:
> Since GDAL already provides the prefix, some of the code in this function could be removed.

yeah, but I wanted to remain as close as the original distutils logic
(follow-up fix for https://github.com/OSGeo/gdal/issues/4516 to come)